File: Data Structures For Interview/Java/Matrix Programs/MultiplyMatrix.java
class MultiplyMatrix {

    static void mul(int m1[][],int m2[][]){
        int m3[][]=new int[3][3];

        for(int i=0;i<3;i++){
            for(int j=0;j<3;j++){
                m3[i][j]=0;
               for(int k=0;k<3;k++){
                m3[i][j]+=m1[i][k]*m2[k][j];
               }
               System.out.print(m3[i][j]+"  ");
            }
            System.out.println();
        }

    }

    public static void main(String s[]){

        int m1[][]={
            {1,1,1},
            {2,2,2},
            {3,3,3}
        };

        int m2[][]={
            {1,1,1},
            {2,2,2},
            {3,3,3}
        };

        mul(m1,m2);

    }
}

File: Data Structures For Interview/JavaScript/Matrix Programs/MultiplyMatrix.js
function mul(m1,m2){

    let m3=[
        [0,0,0],
        [0,0,0],
        [0,0,0]
    ];

    for(let i=0;i<m1.length;i++){
        for(let j=0;j<m2[0].length;j++){
            for(let k=0;k<m2.length;k++){
                m3[i][j]+=m1[i][k]*m2[k][j];
            }
        }
    }
    
    console.log(m3);

}

let m1=[
    [1,2,3],
    [4,5,6],
    [7,8,9]
];

let m2=[
    [1,2,3],
    [4,5,6],
    [7,8,9]
];

mul(m1,m2)